app/templates/import_holidays.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+
+<form class="m-3" action="update_holidays" method="post" enctype="multipart/form-data">
+    <h5 class="title">Import holidays using ics file</h5>
+    <h6 class="title">Use the upload button to choose ics file</h6>
```

Comment:
Why `h6` btw? Is that a 6th level title?

Reply:
Used it just for font size of the heading.

Reply:
Use CSS for this. h5/h6 has semantic meaning of a title :)